### PR TITLE
added libcaer to humble/iron/rolling

### PR DESCRIPTION
**Pull request by berndpfrommer**
# Please Add This Package to be indexed in the rosdistro.
package name: libcaer
distros: humble, iron, rolling

# The source is here:
https://github.com/ros-event-camera/libcaer

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: humble/distribution.yaml [hunk 1]
```diff
@@ -2978,6 +2978,16 @@ repositories:
       type: git
       url: https://github.com/lgsvl/lgsvl_msgs.git
       version: foxy-devel
+  libcaer:
```
Comment (clalancette) [inline humble/distribution.yaml:2981]: Can you please explain the relationship between `libcaer` here and `libcaer_driver` below?

Comment (berndpfrommer) [reply humble/distribution.yaml:2981]: libcaer_driver depends on libcaer (a library that provides access to all of Inilab's devices).
I first tried to compile and install libcaer directly inside the libcaer_driver workspace but couldn't get it to work so I decided to provide it as a separate package. That will also make it easier if somebody wants to access libcaer directly for other ROS projects.

**Review by clalancette — APPROVED** [review]
This looks OK to me, but we can't merge it because `iron` is in a sync-hold.

@audrow I'd appreciate a look from you as well.

**Comment by berndpfrommer** [comment]
Is some input missing from my side for this PR to move forward? I think iron is no longer in sync hold.

**Pull request merged**